lib/use_cases/administrator/get_logs_for_username.rb

```diff
@@ -0,0 +1,13 @@
+module UseCases
+  module Administrator
+    class GetLogsForUsername
+      def initialize(gateway:)
```

Comment:
Even when there's only one `gateway` I'd use a more descriptive name for the named parameter - maybe `log_gateway`, or even `auth_request_gateway`.

Been going back and forth on this naming in `logging` as well, feels like it's more important to talk about WHAT we're logging, `log` is very generic.

Maybe this reflects on the name for the whole use case and it should be `ViewAuthRequestsForUsername`?